guidelines/enhancement_template.md

```diff
@@ -23,6 +24,9 @@ superseded-by:
 
 Start by filling out this header template with metadata for this enhancement.
 
+* Enhancements should have agreement from all stakeholders prior to merging and
+  target implementation in the near future.
```

Comment:
This reads a little choppy. What do you think about splitting it into 2 items like this:

```suggestion
* Enhancements should be related to work to be implemented in the near future.

* Enhancements should have agreement from all stakeholders prior to merging.
```

Reply:
Good call - done!